bindings/pyroot_experimental/PyROOT/src/TDirectoryFilePyz.cxx

```diff
@@ -0,0 +1,71 @@
+// Author: Massimiliano Galli CERN  08/2019
+// Original PyROOT code by Wim Lavrijsen, LBL
+
+/*************************************************************************
+ * Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include "CPyCppyy.h"
+#include "CPPInstance.h"
+#include "CPPOverload.h"
+#include "PyROOTPythonize.h"
+#include "ProxyWrappers.h"
+#include "Python.h"
+#include "Utility.h"
+#include "PyzCppHelpers.hxx"
+#include "TClass.h"
+#include "TKey.h"
+#include "TDirectoryFile.h"
+
+using namespace CPyCppyy;
+
+////////////////////////////////////////////////////////////////////////////
+/// \brief Allow access to objects through the method Get()
+/// This concerns both TDirectoryFile and TFile, since the latter
+/// inherits the Get method from the former.
+/// We decided not to inject this behavior directly in TDirectory in order
```

Comment:
I would say the reason to not inject it in TDirectory is different: TDirectory already has a templated method Get that, when invoked from Python, returns an object of the derived class (e.g. TH1F), and not a generic TObject.